src/applications/static-pages/sass/modules/_m-checklist-layout.scss

```diff
@@ -0,0 +1,7 @@
+.va-l-detail-page--checklist {
+  @media print {
+    #resources-search-bar-wrapper {
+      display: none;
+    }
+  }
+}
```

Comment:
Seems kinda silly to have this in its own file. I expected it to be bigger than this. Would it be better off somewhere else?